### ISSUE TITLE
macOS build fails with undeclared identifier HW_Flag_BaseTypeFromArg

### ISSUE THREAD

**Issue by jbcoe** [dotnet/coreclr#16722]
building on macOS (High Sierra) using the build.sh script leads to the following error: 

```
coreclr-github/src/jit/hwintrinsiclistxarch.h:197:382: error: use of undeclared identifier 'HW_Flag_BaseTypeFromArg'; did you mean 'HW_Flag_BaseTypeFromFirstArg'?
  ...INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid},           HW_Category_MemoryStore,                       HW_Flag_BaseTypeFromArg|HW_Flag_NoRMWSeman...
                                                                                                                                        ^~~~~~~~~~~~~~~~~~~~~~~
                                                                                                                                        HW_Flag_BaseTypeFromFirstArg
/Users/jon/DEV/coreclr-github/src/jit/hwintrinsicxarch.cpp:24:114: note: expanded from macro 'HARDWARE_INTRINSIC'
    {NI_##id, name, InstructionSet_##isa, ival, size, numarg, t1, t2, t3, t4, t5, t6, t7, t8, t9, t10, category, flag},
                                                                                                                 ^
/Users/jon/DEV/coreclr-github/src/jit/namedintrinsiclist.h:79:5: note: 'HW_Flag_BaseTypeFromFirstArg' declared here
    HW_Flag_BaseTypeFromFirstArg = 0x400,
    ^
1 error generated.
```

**Comment by RussKeldorph** [comment]
Please close if fixed by 41f9ab8d44da435de5c0db3855eb8700fdc9a8a1.  @fiigii

**Comment by jbcoe** [comment]
Fixed. Thanks for the fast response.
